### PR TITLE
Add deprecation notice

### PR DESCRIPTION
**Pull request by WadeBarnes**

### CODE REVIEW

**Review by ryjones — APPROVED** [review]
Should the repo be marked read-only as well?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+### [indy-vdr](https://github.com/hyperledger/indy-vdr)
+A module for interacting with the [Indy Node](https://github.com/hyperledger/indy-node) ledger.
+
+### [aries-askar](https://github.com/hyperledger/aries-askar)
+A secure (encrypted at rest) storage and a key management service suitable for use with [Hyperledger Aries] agents and possibly other digital trust agents.  Askar is a replacement implementation (with lessons learned!) of the [indy-wallet] part of the [Hyperledger Indy SDK]. Askar has been demonstrated to be more performant and stable than the Indy SDK when under comparable load.
+
+### [indy-shared-rs](https://github.com/hyperledger/indy-shared-rs)
+Contains the following shared Rust libraries for Hyperledger Indy
+- `indy-credx`: Indy verifiable credential issuance and presentation (aka Anoncreds).
+- `indy-data-types`: Data type definitions for Schemas, Credential Definitions and other types related to credential issuance and processing.
```
Comment (TimoGlastra) [inline README.md:16]: This should point to the AnonCreds RS repository I think. It supports what indy-shared-rs supports + ledger agnostic anoncreds.

https://github.com/hyperledger/anoncreds-rs

This is also where the work will be happening for making AnonCreds W3C data model compliant

**Review by TimoGlastra — APPROVED** [review]
One comment about the anoncreds implementation, otherwise 👍  from me on marking indy-sdk as deprecated

File: README.md (hunk 1 above)
Comment (dhh1128) [reply README.md:16]: Especially if we are going to mark the repo as read-only, as @WadeBarnes and @ryjones were discussing, we should update the links as @TimoGlastra recommends.

Comment (WadeBarnes) [reply README.md:16]: That's the sort of feedback I was looking for.  Thanks!

**Comment by WadeBarnes** [comment]
@TimoGlastra, @dhh1128, I've replaced the `indy-shared-rs` section with `anoncreds-rs` as suggested.  Please have another quick look.

**Comment by TimoGlastra** [comment]
Changes look good to me!

**Comment by dhh1128** [comment]
LGTM

**Pull request merged**